.github/actions/create-jira-issue/entrypoint.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+sprint=$(curl -u ${JIRA_AUTH} -X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/agile/1.0/board/1/sprint?state=active | jq '.values[].id')
+
+if [ ! -z "$sprint" ]
+then
+sprint_id=',"customfield_10008": '${sprint}
+fi
```

Comment:
```suggestion
[[ -n "${sprint_id}" ]] && sprint=',"customfield_10008": '${sprint_id}
```

Reply:
@tei-k san,
For this change, do we need to change the first line of the script to `#!/bin/bash`?. 
Executing with `#!/bin/sh` not working for me. 
Getting error `[[: not found`.
Reference: https://stackoverflow.com/questions/3401183/bash-syntax-error-not-found

Reply:
@mathewseby 

What shell do you use?
I've confirmed that it works fine with `zsh` and `bash` and `busybox`.

Reply:
@tei-k san,
> What shell do you use?

I am using `bash` shell.

Reply:
What is the version?

My local version.
```
$ /bin/sh --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin20)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Or try the following.
```suggestion
if [ -n "${sprint_id}" ]; then
  sprint_id=',"customfield_10008": '${sprint}
fi
```

Reply:
> What is the version?

Following is the version
```
$ /bin/bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
```

> Or try the following.

This one is working.